components/application-operator/charts/application/templates/policy.yaml

```diff
@@ -7,4 +8,5 @@ spec:
     - name: {{ .Release.Name }}-event-service
   peers:
     - mtls:
-        mode: "PERMISSIVE"
\ No newline at end of file
+        mode: "PERMISSIVE"
```

Comment:
this policy should not be needed anymore due to the fix with the host header

Reply:
Right, policy is not needed for non-strict mode as well.